include/mlir/Support/VulkanRuntime.h

```diff
@@ -0,0 +1,59 @@
+//===- VulkanRuntime.h - MLIR Vulkan runtime ------------------------------===//
```

Comment:
This file should be in /Support.

Reply:
@River707 thanks, I'll delete it, since it's just for the test purpose, I think I can declare the function inside the test file.